card/src/main/java/com/adyen/checkout/card/data/CardType.kt

```diff
@@ -7,10 +7,52 @@
  */
 package com.adyen.checkout.card.data
 
+import android.os.Parcelable
+import kotlinx.parcelize.IgnoredOnParcel
+import kotlinx.parcelize.Parcelize
 import java.util.regex.Pattern
 
+@Parcelize
+data class CardType(val txVariant: String) : Parcelable {
+
+    @IgnoredOnParcel
+    private var cardBrand: CardBrand? = null
+
+    /**
+     * Use this constructor when defining the supported card brand predefined inside [CardBrand] enum
+     * inside your component
+     */
+    constructor(cardBrand: CardBrand) : this(txVariant = cardBrand.txVariant) {
+        this.cardBrand = cardBrand
+    }
+
+    companion object {
+        /**
+         * Estimate all potential [CardTypes][CardType] for a given card number.
+         *
+         * @param cardNumber The potential card number.
+         * @return All matching [CardTypes][CardType] if the number was valid, otherwise an empty [List].
+         */
+        fun estimate(cardNumber: String): List<CardType> {
+            return CardBrand.values().filter { it.isEstimateFor(cardNumber) }.map { CardType(cardBrand = it) }
+        }
+
+        /**
+         * Get CardType from the brand name as it appears in the Checkout API.
+         */
+        fun getByBrandName(brand: String): CardType {
```

Comment:
We don't need this method anymore, the constructor with `txVariant` should replace it

Reply:
We shouldn't remove the functionality completely, I meant it should still be done but somewhere else, in the constructor with `txVariant`